### PR TITLE
feat: update @here-wallet/core

### PR DESCRIPTION
**Pull request by AZbang**
@here-wallet/core updated to latest version

P.S. HERE/HOT Wallet does not support testnet

@race-of-sloths

### CODE REVIEW

**Comment by frol** [comment]
Happy to see this happening!

@AZbang I'd like to invite you to participate in the [Race of Sloths](https://race-of-sloths.com/). Just mention @race-of-sloths user in your GitHub comment or PR description to join!

**Comment by AZbang** [comment]
@race-of-sloths

**Comment by race-of-sloths** [comment]
@AZbang Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
New Sloth joined the Race! Welcome!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/AZbang?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/AZbang?type=bot&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/AZbang)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/AZbang)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by race-of-sloths** [comment]
### ❓Unknown command 
Please check the command and try again

**Review by pivanov — APPROVED** [review]
Hey @AZbang 
Thanks for the quick update!!!
I went ahead and made [commit](https://github.com/near/wallet-selector/pull/1155/commits/b2a6b29544bb51b80a612d59c51a9377a52e955c) to fix some lint errors.
Hope that's okay!

File: packages/here-wallet/src/lib/selector.ts
Comment (pivanov) [inline packages/here-wallet/src/lib/selector.ts:12]: Maybe we should leave the [default URL](https://my.herewallet.app/connector/index.html)?

**Review by gtsonevv — DISMISSED** [review]

File: packages/here-wallet/src/lib/selector.ts
Comment (AZbang) [reply packages/here-wallet/src/lib/selector.ts]: yes, my bad

**Review by pivanov — DISMISSED** [review]
r+

**Review by pivanov — APPROVED** [review]

**Review by gtsonevv — APPROVED** [review]

**Comment by race-of-sloths** [comment]
### 🔄 The PR has been merged
@pivanov, please score the PR with `@race-of-sloths score [1/2/3/5/8/13]`. The contributor deserved it
As an alternative option auto-scoring [1, 2] will be applied to this PR!

**Pull request merged**
